.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,106 @@
+# Contributing
+
+I appreciate your considering contributing `eslint-config-moneyforward`. This document is a guide to help make your contribution easier.
+
+## Getting Started
+
+### Prerequisites
+
+- [Node.js](https://nodejs.org) (See [`.node-version`](/moneyforward/eslint-config-moneyforward/.node-version))
```

Comment:
The link for `.node-version` seems wrong. I moved to https://github.com/moneyforward/eslint-config-moneyforward/blob/docs/contribute-guide/moneyforward/eslint-config-moneyforward/.node-version on click the link.

<img width="2233" alt="image" src="https://github.com/moneyforward/eslint-config-moneyforward/assets/40013676/7375ecc2-f25d-4dae-8ac5-b0ee73402ae2">

Fix the link like my suggestion or replace it with the completed URL instead.

```suggestion
- [Node.js](https://nodejs.org) (See [`.node-version`](/.node-version))
```

Reply:
Some other links to this repository (specified like `moneyforward/eslint-config-moneyforward/*`) also seem to jump to the wrong pages.

Reply:
fixed: [436e0ce](https://github.com/moneyforward/eslint-config-moneyforward/pull/304/commits/436e0ce4622ba16cccf15f7f2723d0fa3bcb049e)